plugins/modules/docker_swarm.py

```diff
@@ -642,6 +661,7 @@ def main():
         ),
         default_addr_pool=dict(docker_py_version='4.0.0', docker_api_version='1.39'),
         subnet_size=dict(docker_py_version='4.0.0', docker_api_version='1.39'),
+        data_path_addr=dict(docker_py_version='4.0.0', docker_api_version='1.31'),
```

Comment:
According Docker SDK for Python, this is supported for 1.30 already:
```suggestion
        data_path_addr=dict(docker_py_version='4.0.0', docker_api_version='1.30'),
```
The API docs also confirm that: https://docs.docker.com/engine/api/v1.30/#operation/SwarmInit https://docs.docker.com/engine/api/v1.30/#operation/SwarmJoin

Reply:
Thanks for pointing it out. My source actually were [commandline docs](https://docs.docker.com/engine/reference/commandline/swarm_init/#usage), which state API version 1.31+.

FYI: docker/docker.github.io#14660